tests/unittests/alertengine/base_test.py

```diff
@@ -0,0 +1,17 @@
+from datetime import time, datetime
+from mock import Mock
+from nav.alertengine.base import _calculate_timeperiod_start
+
+
+def test_calculate_timeperiod_start_should_detect_period_starting_yesterday():
+    mock_now = datetime(year=2020, month=8, day=1, hour=7, minute=50)
```

Comment:
No timezone-support, NAV runs with naive datetimes?

Argus's support for infinity in datetimes actually works with timezones.

Reply:
> No timezone-support, NAV runs with naive datetimes?

Yes, it's a poor design decision from two decades ago 😛 🤦